spec/requests/api/v3/activity_resource_spec.rb

```diff
@@ -80,9 +84,40 @@
       end
     end
 
-    it_behaves_like 'handling anonymous user', 'Activity', '/api/v3/activities/%s' do
+    it_behaves_like 'handling anonymous user' do
       let(:project) { FactoryGirl.create(:project, is_public: true) }
-      let(:id) { activity.id }
+      let(:path) { api_v3_paths.activity activity.id }
+    end
+  end
+
+  describe '#patch' do
+    subject(:response) { last_response }
+    let(:patch_path) { api_v3_paths.activity activity.id }
```

Comment:
I actually thought that it is how all specs do it and that is why I sticked to it.

Personally I like the generic `path` more. You won't issue two different requests in one spec anyway.